src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -46,6 +46,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <EnableUnsafeBinaryFormatterSerialization Condition="'$(EnableUnsafeBinaryFormatterSerialization)' == ''">false</EnableUnsafeBinaryFormatterSerialization>
   </PropertyGroup>
 
+
+  <PropertyGroup Condition="'$(PublishTrimmed)' == 'true' Or '$(IsTrimmable)' == 'true' Or '$(PublishAot)' == 'true'">
```

Comment:
`IsTrimmable` is more for libraries. Apps don't ever set this property. I think we should only default this value when `PublishTrimmed` or `PublishAot` == true.